src/utils/common.ts

```diff
@@ -256,20 +252,24 @@ export async function estimateTotalGasRequiredByUnsignedTransaction(
   options: Partial<{
     gasPrice: BigNumberish;
     gasUnits: BigNumberish;
+    baseFeeMultiplier: number;
     transport: Transport;
   }> = {}
 ): Promise<TransactionCostEstimate> {
-  const { gasPrice: _gasPrice, gasUnits, transport } = options || {};
+  const { gasPrice: _gasPrice, gasUnits, baseFeeMultiplier: _baseFeeMultiplier, transport } = options || {};
+  const baseFeeMultiplier = _baseFeeMultiplier ?? 1.0; // A default 1.0x multiplier means no multiplier by default.
```

Comment:
We can get away with one line here :+1: 

```suggestion
  const { gasPrice: _gasPrice, gasUnits, baseFeeMultiplier = 1.0, transport } = options || {};
```